fearless/Modules/Staking/ControllerAccount/ControllerAccountInteractor.swift

```diff
@@ -80,6 +81,14 @@ extension ControllerAccountInteractor: ControllerAccountInteractorInputProtocol
         }
     }
 
+    func fetchControllerAccountInfo(controllerAddress: AccountAddress) {
+        clear(dataProvider: &controllerAccountInfoProvider)
+        controllerAccountInfoProvider = subscribeToAccountInfoProvider(
```

Comment:
One this which is I just noticed is that subscription just retrieves data from database. We need here the same approach as for ledger - just fetch from blockchain